assets/src/scripts/accessibility.js

```diff
@@ -8,7 +6,7 @@ const { select } = require('d3-selection');
  * @param {String} description
  * @param {Boolean} isInteractive
  */
-function addSVGAccessibility({svg, title, description, isInteractive}) {
+function addSVGAccessibility(svg, {title, description, isInteractive}) {
```

Comment:
I'm still not sure when it's appropriate to include an argument in a destructuring target. What's the rationale for this change?

Reply:
When you use `d3.call`, it passes the current element as the first argument. Also, the `link` function will pass the additional arguments (derived from the selector it accepts) as the second argument... so the convention for our D3 "component" functions would be `(elem, options)`.

Reply:
That makes a lot of sense.